### DISCUSSION TITLE
Activate Logger On Android Kiwi & Firefox Fenix

### DISCUSSION THREAD

**Discussion by blogmangit** [uBlockOrigin/uBlock-issues#1879] [Show and tell] [answered]
Logger is by default disabled in ublock origin mobile version for both chrome and Firefox base but we can enable it just do the following: 

1. ublock origin > settings
2.  tick I am an advanced user then enter its settings
3. set UiPopupConfig to +logger
4. apply restart browser
5. logger option is back in mobile.

Regards. 

![16_12_2021_00_16_35.jpg](https://user-images.githubusercontent.com/92579403/146255021-2794eb2c-1e0e-408d-a692-8967eabb89f4.jpg)

![16_12_2021_00_23_15.jpg](https://user-images.githubusercontent.com/92579403/146255038-8cbedc8c-d0dd-4d94-b756-4b1ab243ccd6.jpg)

**Comment by mpeter50** [comment]
This still works, but just discovered that it is not needed actually.

If you dont do this modification, but just open the uBlock settings (also called dashboard), and switch to the "Support" tab there, there is a colored "The logger" link, which opens the logger when clicked.
After this you want to select the tab you want to view in the dropdown list at the top, and done. Press the reload button besides it to reload the page and see the logs.

Bonus is that the DOM inspector works here too. Its much harder to use than on PC because only one browser tab/window can be visible at a time on mobile, but its functional.

**Reply by u-RraaLL** [reply, marked as answer]
And once the logger is open you can bookmark it for easier access.

That was the suggestion before the setting was added.

**Reply by garry-ut99** [reply]
No need to discover it, just read the documentation, everything is already mentioned in documentation:
- https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#open-the-logger
- https://github.com/gorhill/uBlock/wiki/The-logger
